### ISSUE TITLE
GridColumnMenu Dropdown opening upwards

### ISSUE THREAD

**Issue by ahan1986** [telerik/kendo-react#579]
I have couple Grids very close to the bottom of the screen. When the screen is small, the Column Menu opens upwards. The issue comes for me when I _then_ click on the 'Filter' or 'Columns', the dropdowns for those open downward, gets longer and goes off screen. Is there a way to customize that so that it also goes up when it hits the bottom of the screen?

![image](https://user-images.githubusercontent.com/28400128/81592678-504ab800-938c-11ea-9f20-77caaea36dd8.png)

**Comment by simonssspirit** [comment]
@ahan1986  the DropDownList should open upwards by default if there is no space:

https://stackblitz.com/edit/react-vibidv?file=app/main.jsx

**Comment by ahan1986** [comment]
Hi,
So I did notice that the columnMenu does open up but my issue is when you further click on 'Filter' or 'Column', the already-opened columnMenu extends downwards making it go off screen.

**Comment by simonssspirit** [comment]
@ahan1986 If I understand correctly the expected result is the ColumnMenu that is already open downwards to flip upwards when expanding the filter section and there is no enough space at the bottom?

cc: @nstoychev

**Comment by ahan1986** [comment]
Correct. Seems like when you further expand the already-opened ColumnMenu, it only expands down. I would like to somehow configure the components such as '**_GridColumnMenuCheckBoxFilter_**' to expand upwards when it's too close to the bottom of the screen.
